sys/hashes/sha384.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2023 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_hashes
+ *
+ * @{
+ * @file
+ * @brief       SHA384 hash function implementation
+ *
+ * @author      Mikolai Gütschow <mikolai.guetschow@tu-dresden.de>
+ *
+ * @}
+ */
+
+#include <string.h>
+
+#include "hashes/sha384.h"
+
+void sha384_init(sha384_context_t *ctx)
```

Comment:
~~Not an expert on sha's; E.g. `sha224.h` defines:~~
```C
void sha224_init(sha224_context_t *ctx);
static inline void sha224_update(sha224_context_t *ctx, const void *data, size_t len);
static inline void sha224_final(sha224_context_t *ctx, void *digest);
void *sha224(const void *data, size_t len, void *digest);
```
~~Similar with sha256.~~

~~Why do we only have two functions for sha384?~~

Confused the *.c with *.h... comment stays for humorous purposes.